portal-kernel/src/com/liferay/portal/kernel/util/ESMURLUtil.java

```diff
@@ -13,6 +13,16 @@
  */
 public class ESMURLUtil {
```

Comment:
I saw last week that we have an `esm` package in `portal-kernel` already. Maybe we can move all the ESM stuff like this there? :thinking: 

This is the package I'm referring to -> https://github.com/liferay/liferay-portal/tree/131714e6b399906ac4008bd6dd029bca0e9d9574/portal-kernel/src/com/liferay/portal/kernel/frontend/esm

It has a source maps related sibling directory too.

Reply:
Oh nice, I hadn't seen that. yeah I will move it into there